### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by arielelkin**
addresses https://github.com/jonschlinkert/sublime-monokai-extended/issues/81 and https://github.com/jonschlinkert/sublime-markdown-extended/issues/134

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,4 +166,4 @@ Released under the [MIT license](LICENSE).
 [soda]: https://github.com/buymeasoda/soda-theme
 [ba]: https://github.com/jonschlinkert/sublime-monokai-extended/issues/4
 [yfm]: http://assemble.io/docs/YAML-front-matter.html "YAML Front Matter"
-[gfm]: https://help.github.com/articles/github-flavored-markdown#syntax-highlighting "GitHub Flavored Markdown"
\ No newline at end of file
+[gfm]: https://help.github.com/articles/github-flavored-markdown#syntax-highlighting "GitHub Flavored Markdown"
```
Comment (blag) [inline README.md]: What is this changing? These two lines look exactly the same to me...

Comment (arielelkin) [inline README.md]: I don't know why it looks like I'm changing this line, but I'm not. See for yourself:

https://raw.githubusercontent.com/jonschlinkert/sublime-monokai-extended/master/README.md

https://raw.githubusercontent.com/arielelkin/sublime-monokai-extended/01c74f92d28af3e8ac0f08cc9387c3a0a7fcb75f/README.md

Comment (blag) [inline README.md]: > \ No newline at end of file

Oh gotcha. GitHub just rendered the exact same line twice. Thanks!

Comment (arielelkin) [inline README.md]: huh? Neither file has a newline at the end!

Comment (blag) [inline README.md]: Then I am thoroughly confused. 😄 

PR looks good otherwise though.

**Comment by jonschlinkert** [comment]
thank you! I really appreciate it!

**Pull request merged**